cmake/i386-linux-gnu.toolchain.cmake

```diff
@@ -1,5 +1,7 @@
 set(CMAKE_SYSTEM_PROCESSOR i386)
 set(CMAKE_SYSTEM_NAME Linux)
 set(CMAKE_SYSTEM_VERSION gnu)
-set(CMAKE_C_COMPILER gcc)
+if(NOT ${CMAKE_C_COMPILER})
+  set(CMAKE_C_COMPILER gcc)
```

Comment:
@jszakmeister I changed this to be able to use a different compiler (in this case, `gcc-5`) to build for 32 bit. `NOT CMAKE_C_COMPILER` and `NOT DEFINED CMAKE_C_COMPILER` both didn't work. Do you think it's okay like this?